libs/sdk/src/lib/kinetic-sdk-internal.ts

```diff
@@ -145,6 +145,10 @@ export class KineticSdkInternal {
 
     const account = await this.getTokenAccounts({ account: destination, mint })
 
+    if (!senderCreate && account?.length === 0) {
+      throw new Error(`Destination account doesn't exist.`)
```

Comment:
Let's move this to the `handleTransaction` method right after `const solana = await this.data.getSolanaConnection(environment, index)`.
`

Reply:
It seems like I would like to give this a little bit more thought, because this logic does't apply straightforward into the backend, since we don't have the pure destination account anymore, but the destinationTokenAccount, so it would be like resolving the Token Account of a Token Account.

Meaning in the API, instead of having `BobQoPqWy5cpFioy1dMTYqNH9WpC39mkAEDJWXECoJ9y` we have `56kVDoZwvy1paDFs7wy4m3dF6jAzAdDKBZuBM3JuKdqG`. 

Another thing, is to decide if we want to pass down to the API the `senderCreate` so we can identify there if it was false or undefined and validate. I've been thinking if we can identify from the tx instructions to check if we don't have a create instruction.  Anyway, I think I am probably  over-engineering this.

Reply:
Ok - that makes sense. For now, let's keep it like this in the SDK. I'll merge this in and make a note that we want to move these things to the server.